tools/buildPackage.js

```diff
@@ -154,10 +192,31 @@ if (isLinux) {
   cmds.push('copy .\\res\\start-tile-150.png "' + path.join(buildDir, 'resources', 'start-tile-150.png') + '"')
   cmds.push('makensis.exe -DARCH=' + arch + ` res/${channel}/braveDefaults.nsi`)
   cmds.push('ncp ./app/extensions ' + path.join(buildDir, 'resources', 'extensions'))
+
   // Make sure the Brave.exe binary is squirrel aware so we get squirrel events and so that Squirrel doesn't auto create shortcuts.
   cmds.push(`"node_modules/rcedit/bin/rcedit.exe" ./${appName}-win32-` + arch + `/${appName}.exe --set-version-string "SquirrelAwareVersion" "1"`)
 }
 
+// Verify tor binaries and bundle with Brave
+var torPath
+if (isDarwin) {
+  torPath = path.join(buildDir, `${appName}.app`, 'Contents', 'Resources', 'extensions', 'bin')
+} else {
+  torPath = path.join(buildDir, 'resources', 'extensions', 'bin')
+}
+
+cmds.push('mkdirp ' + torPath)
+cmds.push('curl -o ' + path.join(torPath, 'tor') + ' ' + torURL)
+cmds.push('curl -o ' + path.join(torPath, 'tor-sig') + ' ' + torSigURL)
+fs.writeFileSync('temp.asc', torVerificationPublicKey, 'utf8')
```

Comment:
you can just check this file into git so it doesn't need to be written/deleted every time